x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -79,33 +103,80 @@ export const links: LinkItem = {
     {
       id: SecurityPageName.policies,
       title: POLICIES,
+      description: i18n.translate('xpack.securitySolution.appLinks.policiesDescription', {
+        defaultMessage:
+          'Use policies to customize endpoint and cloud workload protections and other configurations.',
+      }),
+      landingIcon: IconEndpointPolicies,
       path: POLICIES_PATH,
       skipUrlState: true,
       experimentalKey: 'policyListEnabled',
     },
     {
       id: SecurityPageName.trustedApps,
       title: TRUSTED_APPLICATIONS,
+      description: i18n.translate(
+        'xpack.securitySolution.appLinks.trustedApplicationsDescription',
+        {
+          defaultMessage:
+            'Improve performance or alleviate conflicts with other applications running on your hosts.',
+        }
+      ),
+      landingIcon: IconTrustedApplications,
       path: TRUSTED_APPS_PATH,
       skipUrlState: true,
     },
     {
       id: SecurityPageName.eventFilters,
       title: EVENT_FILTERS,
+      description: i18n.translate('xpack.securitySolution.appLinks.eventFiltersDescription', {
+        defaultMessage: 'Exclude unwanted applications from running on your hosts.',
+      }),
+      landingIcon: IconEventFilters,
       path: EVENT_FILTERS_PATH,
       skipUrlState: true,
     },
     {
       id: SecurityPageName.hostIsolationExceptions,
       title: HOST_ISOLATION_EXCEPTIONS,
+      description: i18n.translate('xpack.securitySolution.appLinks.hostIsolationDescription', {
+        defaultMessage: 'Allow isolated hosts to communicate with specific IPs.',
+      }),
+      landingIcon: IconHostIsolation,
       path: HOST_ISOLATION_EXCEPTIONS_PATH,
       skipUrlState: true,
     },
     {
       id: SecurityPageName.blocklist,
       title: BLOCKLIST,
+      description: i18n.translate('xpack.securitySolution.appLinks.blocklistDescription', {
+        defaultMessage: 'Exclude high volume or unwanted events being written into Elasticsearch.',
```

Comment:
and this message should be for `eventFilters`. Same as @dasansol92 pointed out. I belive the mock swapped the messages. Here is a screenshot of existing blocklist list page.

![Screenshot 2022-05-17 at 10 22 55](https://user-images.githubusercontent.com/1849116/168765288-51d3f6d7-d9ff-4cd7-a276-7fc897aafb07.png)